src/controller/CHIPDevice.cpp

```diff
@@ -175,8 +175,19 @@ CHIP_ERROR Device::Deserialize(const SerializedDevice & input)
     return error;
 }
 
+void Device::OnNewConnection(SecureSessionHandle session, SecureSessionMgr * mgr)
+{
+    mState         = ConnectionState::SecureConnected;
+    mSecureSession = session;
+}
+
+void Device::OnConnectionExpired(SecureSessionHandle session, SecureSessionMgr * mgr)
+{
+    mState = ConnectionState::NotConnected;
```

Comment:
Should `mSecureSession` be reset in some way?

Reply:
done